visualization/react-lib/src/components/space-chart/__tests__/SpaceChart.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import SpaceChart, { SpaceChartProps } from "../SpaceChart";
+import { create } from "react-test-renderer";
+// TO DO: Consider using react-faux-dom or jsdom to test d3 //
+// TO DO: Make SpaceChart work by mocking DonutChart. OR possibly this will be solved using react-faux-dom or jsdom. //
+
+// Currently fails
+type Data = {
+  title: string;
+  legendTitle: string;
+  value: number;
+};
+
+interface DonutChartProps {
+  data: Data[];
+  width: number;
+  height: number;
+  innerRadius: number;
+  outerRadius: number;
+  translationValX: number;
+  translationValY: number;
+}
+
+const data1: Data = {
+  legendTitle: "Test",
+  title: "Test",
+  value: 3,
+};
+const dataArr: Data[] = [data1];
+
+const donutProps: DonutChartProps = {
+  data: dataArr,
+  width: 1000,
+  height: 1000,
+  innerRadius: 5,
+  outerRadius: 10,
+  translationValX: 2,
+  translationValY: 1,
+};
+
+describe("Space chart tests", () => {
+  it("Verify SpaceChart", () => {
```

Comment:
Let's fix test and add  another one for time chart

Reply:
fixed the donut chart and line chart tests, will add other tests in another pr